src/TransponderSquawk.hpp

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include <cassert>
+#include <cstdint>
+
+/**
+ * This class stores a transponder squawk
+ */
+class TransponderSquawk {
```

Comment:
I'd like to question the nomenclature. What is a "squawk"? On radio, "squawk" is a command, an abbreviation for "switch to transponder code". But the code itself is not a "squawk" but a "code". But maybe I'm wrong, I've never owned or operated a transponder.

Reply:
Correct. "Squawk" is just a word commonly used in aviation language _(while talking to an ATC)_ to identify an action necessary to change the transponder code.
IMHO, use of  _TransponderCode_ is better to name a class or a file rather than _TransponderSquawk_.

Reply:
Fine for me, will change.